README.md

```diff
@@ -44,6 +44,15 @@ Alternatively, there can be contributions made that aren't code related.
 - General question, gotchas and other discussions [here](https://github.com/AndroidDev-social/MastodonCompose/discussions/categories/general)
 - Create new Issues/Features for post MVP. (e.g. Material You support, animations, V2 functionality, migrations, etc)
 
+
+### Detekt
```

Comment:
I would omit developer infos in the readme.
It is fine to have it in the contribution file only

Reply:
So I should remove that whole block?

Reply:
yes please. If necessary move it to the contribution.md

The Readme is also for non-dev people. They don't care about these technical details